client/src/actions/actionCreator.js

```diff
@@ -0,0 +1,423 @@
+import * as req from 'superagent';
```

Comment:
You can break this file down into multiple files say `documentActions`, `userActions` etc and use this to combine them